app/views/grant_permission_mailers/submission_mailer/submitted_notification.html.haml

```diff
@@ -8,6 +8,9 @@
   Submitter:
   = full_name(@submitter)
   %br
+  Applicants:
+  = full_name_list(@applicants)
+  %br
```

Comment:
Still don't think the Submitter and Applicant need to be included if they are equal.

For the label, you can pass the count of `@applicants` to `.pluralize` to switch between "Applicant" and "Applicants". See [rails docs](https://devdocs.io/rails~6.1/string#method-i-pluralize)

Reply:
@lmkw John and I have a disagreement here. 
This is an email that goes to grant admins when there is a submission. Should this have the submitters and applicants listed in the email? I am of the camp that we should either always include the submitters or always leave them out. John would like to only include the submitters when different from the applicant. Thoughts?

Reply:
While I see your point about consistency, Matt, I also think that consistency can sometimes lead to visual noise. So we should prioritize the applicant name and only show the submitter when it differs from the applicant. 

This is easier to read:

**Applicant**: Dr. Bunsen Honeydew   **Submitter**: Beaker
**Applicant**: Kermit
**Applicant**: Rowlf
**Applicant**: Animal **Submitter**: Janice


than this:
**Applicant**: Dr. Bunsen Honeydew   **Submitter**: Beaker
**Applicant**: Kermit **Submitter**: Kermit
**Applicant**: Rowlf  **Submitter**: Rowlf
**Applicant**: Animal **Submitter**: Janice